docs/library/gc.rst

```diff
@@ -31,7 +29,7 @@ Functions
    .. admonition:: Difference to CPython
       :class: attention
 
-      This function is MicroPython extension.
+      This function is a MicroPython extension.
```

Comment:
Lets not call `gc` supported until we move the extensions out of here.

Reply:
"The functionality of `gc` may change in the future to more closely match that of CPython."